kernels/test/op_cdist_forward_test.cpp

```diff
@@ -40,89 +40,105 @@ class OpCdistForwardOutTest : public ::testing::Test {
+         1.73205078, 6.40312433,  7.48331499,  0.00000000, 6.16441393,
+         5.38516474, 7.81024981,  4.24264050,  6.16441393, 0.00000000});
+    op_cdist_forward_out(x1, x2, 2.0, compute_mode, out);
+    EXPECT_TENSOR_CLOSE(out, l2);
 
-  Tensor l2 = tfFloat.make(
-      {2, 2, 4, 5},
-      {0.00000000, 7.07106781,  8.06225777,  4.12310553, 4.12310553,
-       5.38516474, 7.00000000,  6.00000000,  6.16441393, 7.48331499,
-       7.07106781, 12.80624866, 5.74456263,  3.00000000, 10.04987526,
-       5.09901953, 5.47722578,  8.77496433,  7.68114567, 6.40312433,
-       4.47213602, 4.24264050,  3.31662488,  5.91608000, 4.12310553,
-       3.00000000, 5.00000000,  7.87400770,  2.44948983, 6.16441393,
-       7.87400770, 10.77032948, 6.40312433,  8.30662346, 3.00000000,
-       4.24264050, 2.44948983,  8.06225777,  4.58257580, 7.68114567,
-       4.24264050, 5.65685415,  10.24695110, 7.81024981, 5.38516474,
-       3.31662488, 8.30662346,  8.36660004,  4.24264050, 4.24264050,
-       5.91608000, 6.40312433,  4.69041586,  6.16441393, 7.07106781,
-       4.12310553, 10.04987526, 5.47722578,  0.00000000, 7.34846926,
-       5.47722578, 0.00000000,  7.28010988,  6.40312433, 7.81024981,
-       5.91608000, 7.28010988,  0.00000000,  7.48331499, 4.24264050,
-       1.73205078, 6.40312433,  7.48331499,  0.00000000, 6.16441393,
-       5.38516474, 7.81024981,  4.24264050,  6.16441393, 0.00000000});
-  op_cdist_forward_out(x1, x2, 2.0, compute_mode, out);
-  EXPECT_TENSOR_CLOSE(out, l2);
+    Tensor l3 = tf.make(
+        {2, 2, 4, 5},
+        {0.00000000, 6.00000000,  7.41079521, 3.50339794, 4.02072573,
+         4.62606478, 7.00000000,  5.14256334, 6.01846170, 6.60385466,
+         6.00000000, 11.47758675, 5.05277443, 2.57128167, 9.28704357,
+         5.01329803, 5.11722994,  7.39863634, 7.18551636, 5.73879337,
+         4.16016769, 4.04124022,  3.07231688, 5.34848118, 3.50339794,
+         2.57128167, 4.49794149,  7.23042679, 2.15443468, 6.01846170,
+         6.99319077, 9.25212955,  6.08220196, 7.45903587, 2.57128167,
+         3.77976322, 2.15443468,  8.00520515, 4.17933941, 7.18551636,
+         4.04124022, 5.03968430,  8.88326645, 6.74599648, 4.62606478,
+         3.07231688, 7.45903587,  7.16609573, 4.04124022, 3.77976322,
+         5.34848118, 6.08220196,  3.95789170, 5.42883539, 6.00000000,
+         3.50339794, 9.00000000,  5.11722994, 0.00000000, 7.06069660,
+         5.11722994, 0.00000000,  7.05400419, 6.08220196, 6.74599648,
+         5.34848118, 7.05400419,  0.00000000, 6.60385466, 4.04124022,
+         1.44224954, 6.08220196,  6.60385466, 0.00000000, 5.42883539,
+         4.62606478, 6.74599648,  4.04124022, 5.42883539, 0.00000000});
+    op_cdist_forward_out(x1, x2, 3.0, compute_mode, out);
+    if (DTYPE == ScalarType::BFloat16) {
+      EXPECT_TENSOR_CLOSE_WITH_TOL(
+          out,
+          l3,
+          1e-2,
+          executorch::runtime::testing::internal::kDefaultBFloat16Atol);
```

Comment:
Why do you need to pass the default atol here?

Reply:
macros don't have default arguments